### PR TITLE
Last pr before 2.1 i hope

### PR DESCRIPTION
**Pull request by NopoTheGamer**

### CODE REVIEW

File: Update Notes/2.1.md [hunk 1]
```diff
@@ -113,14 +113,14 @@
 - Added power stone feature - hannibal2
 - Added abiphone warning - hannibal2
 - Added blur limit at 100 - nopo
-- Added bazaar sacks profit feature - hannibal2
```
Comment (hannibal002) [inline Update Notes/2.1.md:116]: looking good

**Review by hannibal002 — APPROVED** [review]

**Comment by Ascynx** [comment]
gaming

**Review by Lulonaut — DISMISSED** [review]

**Comment by jani270** [comment]
🙏

**Review by romangraef — APPROVED** [review]

**Review by hannibal002 — APPROVED** [review]
I dont understand why but ok.

**Comment by jani270** [comment]
> I dont understand why but ok.

You can look in contrib chat

**Review by Lulonaut — APPROVED** [review]

**Pull request merged**